src/foolscap/remoteinterface.py

```diff
@@ -28,21 +29,21 @@ class RemoteInterfaceClass(interface.InterfaceClass):
 
     """
 
-    def __init__(self, iname, bases=(), attrs=None, __module__=None):
+    def __init__(self, iname, bases=(), attrs=None):
         if attrs is None:
-            interface.InterfaceClass.__init__(self, iname, bases, attrs,
-                                              __module__)
+            interface.InterfaceClass.__init__(self, iname, bases, attrs)
             return
 
+        attrs.pop('__metaclass__', None)
```

Comment:
I couldn't figure out why this is necessary or desirable.  I couldn't even find the case where `__metaclass__` is present in attrs.  If there's some case where this matters, I suppose a new test demonstrating it would be valuable.

Reply:
According to [this comment](https://github.com/warner/foolscap/issues/75#issuecomment-708062519) and the one before it, I had to remove it from attrs to allow tests to pass on Python 2.

On further review, it seems that the subsequent commit obviated the need for this workaround and I've removed it in a57be96.